components/LeagueCard/LeagueCard.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { cn } from '@/utils/utils';
+import Image from 'next/image';
+import placeholderImage from './placeholderImage.svg';
+
+interface ILeagueCardProps extends React.HTMLAttributes<HTMLDivElement> {
+  isEliminated?: boolean;
+}
+
+const LeagueCard = React.forwardRef<HTMLDivElement, ILeagueCardProps>(
+  ({ className, isEliminated = false, ...props }, ref) => (
+    <div
+      data-testid="LeagueCard"
+      ref={ref}
+      className={cn(
+        'flex h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800',
+        className,
+        { 'LeagueCardEliminated opacity-50 dark:bg-zinc-700': isEliminated },
+      )}
+      {...props}
+    />
+  ),
+);
+LeagueCard.displayName = 'LeagueCard';
+
+const LeagueCardHeader = React.forwardRef<
```

Comment:
What does Header do in this context? We already have a title. Titles are headings.

Reply:
@shashilo As `LeagueCard`, and all the other `LeagueCard` components, are based on shadcn/ui's `Card` component, the `LeagueCardHeader` is a container for `LeagueCardTitle` and `LeagueCardDescription`

Again, I was following the structure of the component we are using as our base.